browser/app/index.js

```diff
@@ -13,44 +13,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+import 'babel-polyfill'
 import './less/main.less'
-
 import React from 'react'
 import ReactDOM from 'react-dom'
 import thunkMiddleware from 'redux-thunk'
 import createStore from 'redux/lib/createStore'
 import applyMiddleware from 'redux/lib/applyMiddleware'
-
```

Comment:
These spaces were added on purpose to help readability, removing them will create a large import blob which would hurt readability. You should revert these changes.